src/views/clusteroverview/OverviewTab/metric-charts-card/utils/utils.ts

```diff
@@ -0,0 +1,46 @@
+import { ChartData } from './hooks/types';
+import { METRICS } from './constants';
+
+export const formatCurrentValue = (value: number, metric: string) => {
+  switch (metric) {
+    case METRICS.VM:
+      return Math.trunc(value);
+    default:
+      return value;
+  }
+};
+
+export const getCurrentValue = (chartData: ChartData, metric: string) =>
+  formatCurrentValue(chartData?.[chartData?.length - 1]?.y, metric);
+
+export const getLabelUnit = (metric, unit) => {
+  switch (metric) {
+    case METRICS.VM:
+      return 'VMs';
+    case METRICS.VCPU_USAGE:
+      return 'vCPU';
+    case METRICS.MEMORY:
+    case METRICS.STORAGE:
+    default:
+      return unit;
+  }
+};
```

Comment:
can be an mapperObj? can also be with proxy to invalid values

Reply:
Fixed!